### DISCUSSION TITLE
LTSP on the server and on the client

### DISCUSSION THREAD

**Discussion by Mogaba** [ltsp/ltsp#812] [Q&A] [answered]
If I understood correctly, `ltsp initrd` packs server's /usr/share/ltsp into ltsp.img and it's unpacked into /usr/share/ltsp on the client. But the client has it's own LTSP that runs at boot, right? So how does this work exactly? Do I need the exact same LTSP scripts on both the server and the client?

**Comment by alkisg** [comment, marked as answer]
The main reason is to be able to boot images that do NOT have LTSP installed, e.g. ubuntu-mate-2204.iso.

The secondary reason is to be able to send newer or customized server-side acripts to override the ones in the image.

All that is implemented with a simple cp -u command.

**Comment by svetoi1985** [comment]
Hello, I can't figure out how to register in ltsp.conf the autorun script from the client to record the name in dns.
